apps/web/src/components/Profile/NftGallery/Gallery.tsx

```diff
@@ -47,23 +62,88 @@ const Gallery: FC<Props> = ({ galleries }) => {
           }
         });
       }
-    } catch {}
+    } catch (error: any) {
```

Comment:
can we use `Error` type? if not working fine with any

Reply:
I tried with `Error` not working !!!